quinn-proto/src/connection/timer.rs

```diff
@@ -18,8 +18,10 @@ pub(crate) enum Timer {
     Close,
     /// When keys are discarded because they should not be needed anymore
     KeyDiscard,
-    /// When to give up on validating a new path to the peer
+    /// When to give up on validating a new path from unintentional migration
```

Comment:
This could also be intentional migration?  RFC9000-style migrations can also be intentional, and still have to be supported in multipath.

Reply:
only wondering if the comment is a little misleading